ghcup

```diff
@@ -58,10 +58,19 @@ VERBOSE=false
 # Whether to force installation and overwrite files.
 FORCE=false
 
+# @VARIABLE: GHCUP_INSTALL_BASE_PREFIX
+# @DESCRIPTION:
+# The main install directory prefix, under which .ghcup
+# directory will be created. This directory is user
+# configurable via the environment variable of the
+# same name. It must be non-empty and the path
+# it points to must exist.
+: "${GHCUP_INSTALL_BASE_PREFIX:=$HOME}"
```

Comment:
Can I suggest replacing this line with the following?
```
GHCUP_INSTALL_BASE_PREFIX="${GHCUP_INSTALL_BASE_PREFIX:-$HOME}"
```
It seems to me less cryptic and more similar in style to other variables definitions

Reply:
This is normal shell syntax